yeat/cli.py

```diff
@@ -101,6 +111,14 @@ def get_parser():
         action="store_true",
         help="construct workflow DAG and print a summary but do not execute",
     )
+    parser.add_argument(
+        "-d",
+        "--downsample",
+        type=int,
+        metavar="D",
+        default=0,
+        help="downsample reads down to the desired number; by default, D=0; when set to default, YEAT will auto downsample; set D=-1 to not downsample",
+    )
```

Comment:
Cool. Three modes:

1. Auto-downsample to 150x coverage (D=0)
2. Downsample to a fixed number of reads D (D>0)
3. Disable downsampling (D<0)